### ISSUE TITLE
Questions about stylized colors

### ISSUE THREAD

**Issue by TachibanaYoshino** [mnicnc404/CartoonGan-tensorflow#13]
I rebuilt the model for training according to the code you implemented. The test results are as follows：
![007_a](https://user-images.githubusercontent.com/36946777/58641729-5045d980-832e-11e9-97cb-5c2f06f2120b.png)
![007_b](https://user-images.githubusercontent.com/36946777/58641730-51770680-832e-11e9-9813-645f8cb48e2d.png)
I am very surprised, why is there such a problem with the color after the stylization? My model implementation is the same as yours. The training set is more than 6,000 photos and more than 3,000 frames in Hayao Miyazaki's movies. The dataset image size is 256x256.

**Comment by mnicnc404** [comment]
It seems to me that the color is not beyond expectation, depending on your dataset. My model trained with Paprika tends to generate reddish images since most images of the dataset are reddish. Your model generates yellowish images for the same reason.
You can try reducing the weight of style loss `--style_lambda`, but by doing so the training process might take a longer time.

**Comment by TachibanaYoshino** [comment]
When I reduce the weight of the style loss, such as 10.0 or 15.0, I find that the generated image has almost no cartoon style, even when training to 100 epcoh.

**Comment by mnicnc404** [comment]
I don't have your dataset so I can only give advices that might not help. If that's the case, increase the content loss lambda and see if it helps.

In fact, in the original CartoonGan paper, they didn't even use style loss. If you really want to train a model that generates images without color distortion, you may set style-lambda to 0 and a larger g_adv_lambda. If that does not help, I suggest consulting the authors of the CartoonGan paper.

Closing the issue since not much help I can provide.
